docs/apis/metrics.md

```diff
@@ -157,6 +157,55 @@ public class MyMapper extends RichMapFunction<Long, Integer> {
 }
 {% endhighlight %}
 
+#### Meter
+
+A `Meter` measures an average throughput. An occurrence of an event can be registered with the `markEvent()` method. Occurrence of multiple events at the same time can be registered with `markEvent(long n)` method.
```

Comment:
I'd like a short descriptive name :) I can't think of one right now though, so if i had to decide i would rather go with `markEvent` than `mark`.